src/components/timeline/Task.vue

```diff
@@ -134,13 +134,19 @@ export default {
       this.updateTask(x, width);
     },
     onDrag(x) {
+      console.log("onDrag ");
       this.updateTask(x, this.width);
       this.dragging = false;
     },
     onDragStart() {
+      console.log("onDragStart ");
       this.dragging = true;
     },
     updateTask(x, width) {
+      console.log("updateTask ");
+      this.task.assignee = Number(sessionStorage.getItem("assigneeID")); //neue AssigneeId wird dem Task zugeordnet
+      this.editTask(this.task);
+      sessionStorage.clear();
```

Comment:
This clears the whole sessionStorage. If any other data was saved in there under a different key, it would be removed as well. This was probably not your intention.

Reply:
Yah, die Zeile 149 kann man löschen. ( Hab ich vergessen >.<)